runtime/src/lib.rs

```diff
@@ -174,8 +174,8 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("template-parachain"),
-    impl_name: create_runtime_str!("template-parachain"),
+    spec_name: create_runtime_str!("gmordie"),
+    impl_name: create_runtime_str!("gmordie"),
```

Comment:
```suggestion
    spec_name: create_runtime_str!("gm_parachain"),
    impl_name: create_runtime_str!("gm_parachain"),
```

Reply:
Changing to gm_parachain as GMorDie is the team DAO, which is a separate entity from the parachain itself.

Reply:
aah sounds good :)